examples/index.html

```diff
@@ -44,19 +44,22 @@
 </form>
 
 <script type="module">
-import autosize from '../dist/textarea-autosize.js'
+(async function() {
+  //const autosize = (await import('../dist/index.js')).default
+  const autosize = (await import('https://cdn.jsdelivr.net/npm/@github/textarea-autosize/dist/index.js')).default
```

Comment:
We typically use `unpkg` for this. is `jsdelivr` better? Should we switch this or the other examples?

Reply:
Switch this one I reckon.

```suggestion
  const autosize = (await import('https://unpkg.com/@github/textarea-autosize@latest/dist/index.js')).default
```